oracle/datadog_checks/oracle/oracle.py

```diff
@@ -164,18 +191,36 @@ def _get_dsn(self):
         except Exception:
             self._connection_errors += 1
             raise ConfigurationError('server needs to be in the <HOST>:<PORT> format, "%s"" provided' % self._server)
-        return cx_Oracle.makedsn(host, port, service_name=self._service)
+
+        if self._protocol == 'TCPS':
```

Comment:
i see `'TCPS'` string reused a few times, can we assign it to `PROTOCOL_TCPS` variable to prevent typo/errors in the future? It's easier to reuse variables. We can update tcp to `PROTOCOL_TCP` and construct the list of protocols with variables instead of strings.